.changelogs/README.md

```diff
@@ -0,0 +1,54 @@
+# Changelogs
+
+This directory includes temporary changelog entries, in the form of simple `.json` files. This was done to prevent merge conflicts when modifying the same `CHANGELOG.md` file in many PR's at once.
+
+
+## Mandatory PR check
+
+Every pull request in this repository requires a new changelog entry to be created in this directory, asserted by a GitHub actions workflow. The workflow will fail in any PR that does not have a new `.changelogs/*.json` file added. If a pushed commit does not have a PR associated with it, the check is skipped entirely.
+
+**To disable this check**, simply add the following string to the **PR description**:
+
+```
+[skip changelog check]
+```
+
+
+## Changelog helper
+
+This repository includes a script that aids in creating new changelog entries and compiling them into the final `CHANGELOG.md` file.
+
+To see the list of commands and their options, run:
+
+```bash
+bin/changelog
+bin/changelog <command> --help
+```
```

Comment:
A question to the team @handsontable/devs: Do we want to hide this as an NPM script? `npm run changelog` for example?

Reasoning: I'm often checking what is possible on the package.json `scripts` list. May forget about a command in `bin/` folder.

Reply:
Great idea. I sometimes use `npm run` to list the available commands (less often bash-completion). I'd add the `npm run changelog` shortcut to the "scripts".

Reply:
https://github.com/handsontable/handsontable/pull/7448/commits/cf76f590824f9d59eb50a3742bb12dfb0d217d8a

Reply:
Should we adjust README to use npm script now?

Reply:
`npm run changelog` makes noise when printing the command it will execute, and will write out a big ugly npm error in case of an exception:

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/7091471/104344053-f2fb9d80-54fc-11eb-930f-a01742f4c4c9.png">

I will personally still always prefer running `bin/changelog` raw. The npm script is good for discoverability, but in the end imo `bin/changelog` would be the more idiomatic usage.

Reply:
@nikersify We're not missing `process.exit(0)` somewhere? On the other hand, this looks like the correct behavior. The command can't be used correctly without arguments, therefore the exit code is `1`.

This will not change the functionality. I think you should merge it already. We can adjust readme or terminal output anytime.

Reply:
No, exit code 1 is correct in case of running the command without arguments, as the error `Not enough non-option arguments: got 0, need at least 1` points out, if you were to run `npm run changelog -- --help` the help would still be printed but with exit code 0.

All I'm saying that `bin/changelog` doesn't show the `npm ERR!` lines on exit code 1, nor the two lines at the start beginning with `>`, which imo is cleaner.

Reply:
Yup, that's what I wrote. I think we agree here and can proceed with the merge as is :)